ERCS/erc-7752.md

```diff
@@ -0,0 +1,937 @@
+---
+eip: 7752
+title: Restricted Security Token
+description: Restricted security token (private equity, debt, and derivative securuties)
```

Comment:
Please avoid restating your title in your description. It doesn't add any useful information. Instead you should expand on the ideas introduced in your title. What is a restricted security token? Who might find it useful? That kind of thing.

For example:

> Represent private equity, debt, and derivative securities as non-fungible tokens.